developers/development/bindings.md

```diff
@@ -1,107 +0,0 @@
----
-layout: developersguide
-title: Developing bindings
----
-
-{% include base.html %}
-
-# Developing a Binding for openHAB
-
-This page describes the necessary steps in order to implement a new binding for openHAB.
-
-For information about code style and naming conventions, please see the [coding guidelines](guidelines.html).
-
-Choose the option "openHAB 2 Add-ons" in [your IDE setup](ide.html), and create a skeleton for your binding.
-To do this, use a command line terminal and go to your Git repository folder under `git/openhab2-addons/addons/binding` and call the script `create_openhab_binding_skeleton.sh` with two arguments.
```

Comment:
I can't find the information about the skeleton script, right? Should it be added to `developers/index.md`? I can add it if this pr is merged.

Reply:
Yes the script wan't ready when I wrote that section.
You can either create a PR towards this branch (preferred) or wait for Yannick to approve the website change so that this PR gets merged first.

Reply:
I'll see what to do. It looks like this pr is ready to be merged. So I'll think I'll wait for that to not hold on this pr any longer.

I've no concrete plans with the skeleton. My first idea was to reinstate what was there with some improvements like adding binding info to the global files. But the java code is still the same. So there's a whole world of opportunities to improve :smile: